src/mechanics/constitutive/EngineeringStrain.h

```diff
@@ -5,9 +5,9 @@
 
 namespace NuTo
 {
```

Comment:
Shouldn't this be moved into a separate integrand lib? Or the other way round, shouldn't we move the whole pde integration/cell features into another directory/lib and reorganize the mechanics subdirectory into a a subrepo?